ideascube/models.py

```diff
@@ -25,8 +25,8 @@ class TimeStampedModel(models.Model):
     An abstract base class model that provides self-
     updating ``created_at`` and ``modified_at`` fields.
     """
-    created_at = models.DateTimeField(auto_now_add=True)
-    modified_at = models.DateTimeField(auto_now=True)
+    created_at = models.DateTimeField(_('creation date'), auto_now_add=True)
+    modified_at = models.DateTimeField(_('modification date'), auto_now=True)
```

Comment:
I'd prefer you pass the verbose name using its keyword argument:

```python
    modified_at = models.DateTimeField(verbose_name=_('modification date'), auto_now=True))
```

Counting on the order in which the kwargs are defined is a terrible idea, as that order could change.

In addition, not all fields take the verbose name as the first argument (for example `ForeignKey`). Explicitly using the kwarg form is just much clearer, and avoids confusion.